sos/report/plugins/mysql.py

```diff
@@ -98,14 +99,25 @@ def setup(self):
 class DebianMysql(Mysql, DebianPlugin, UbuntuPlugin):
 
     packages = (
-        'mysql-server',
+        'mysql-server.*',
         'mysql-common',
-        'mariadb-server',
-        'mariadb-common'
+        'mariadb-server.*',
+        'mariadb-common',
+        'percona-xtradb-cluster-server-.*',
     )
 
     def setup(self):
         super(DebianMysql, self).setup()
-        self.add_copy_spec("/etc/mysql/conf.d/mysql*")
+        self.add_copy_spec([
+            "/etc/mysql/",
+            "/var/log/mysql/error.log",
+            "/var/lib/mysql/*.err",
+            "/var/lib/percona-xtradb-cluster/*.err",
+            "/var/lib/percona-xtradb-cluster/grastate.dat",
+            "/var/lib/percona-xtradb-cluster/gvwstate.dat",
+            "/var/lib/percona-xtradb-cluster/innobackup.*.log",
+        ])
+        self.add_cmd_output("du -s /var/lib/percona-xtradb-cluster/*")
```

Comment:
This looks like it would be better suited for a new `percona` (or `percona_xtradb` if you prefer) plugin.

Reply:
Ultimately any variant stores pretty much the same data and files, just in two possible data directory locations. So a separate plugin would duplicate basically everything except for the data directory path, so that seemed extreme.

I'd prefer to keep a single plugin but I guess I could in theory try to auto detect the data directory path, however, even some versions of percona exist in /var/lib/mysql so this would mostly end up being a stat() on the directory paths and i'm not sure there's ultimately much of a performance win versus just asking for all the various files. do you have any thoughts/precedent for that?

Reply:
Ack, here is fine.